python/llm/example/CPU/HF-Transformers-Agent/Model/README.md

```diff
@@ -0,0 +1,19 @@
+# BigDL-LLM Transformers INT4 Optimization for HuggingFace Transformers Agent
+You can use BigDL-LLM to run Huggingface Transformers Agent using models with INT4 optimizations on either servers or laptops. This directory contains example scripts to help you quickly get started using BigDL-LLM to run some popular open-source models in the community. Each model has its own dedicated folder, where you can find detailed instructions on how to install and run it.
```

Comment:
I think this is too complicated right now. Let's just make it a simple agent example using Vicuna as a reference model.

Reply:
put this to `example/CPU/Application/hf-agent`. No need to write Application/README.md. We will take care of it later. 
`example/CPU/Application/hf-agent` just contain README, demo.jpg, run_agent.py

Reply:
add the reference COCO image link to the demo image in README.

Reply:
File structure changed and reference COCO image link is added. Thanks!